### ISSUE TITLE
CreateObject: Got unsupported type 0x27

### ISSUE THREAD

**Issue by ya0pengcheng** [zrax/pycdc#494]
First, I started a python3.11.6 docker container in /scripts/Dockerfile.pybuild
Then cmake. & & make installed successfully
Finally, use pycdc to reverse the error in the compile.pyc file
If pycdc does not have the -c -v parameter, Unsupported opcode: RETURN_GENERATOR will appear
May I ask what parameters I need to include when compiling, or if there are other issues? Thank you for your help:)

<img width="1005" alt="Screen Shot 2024-06-26 at 15 05 05" src="https://github.com/zrax/pycdc/assets/167167390/675c354c-4fc4-4600-b80c-82f83012c25a">

**Comment by greenozon** [comment]
yes, there are other issues
one need to add support (code) the missed functionality

**Comment by ya0pengcheng** [comment]
> yes, there are other issues one need to add support (code) the missed functionality

yeah, 

> opcode

Yeah, thanks for your reply:).
Yesterday, I tried to add Lack's opcode to the repository under fork. I found that there were many missing opcodes in the asynchronous part of Python 3.11, which were only defined in the Header File and not implemented in terms of functionality. I felt that it might take a long time, so I switched the solution and closed the issue.
I hope it can be improved in the future. I really like this project.
